### ISSUE TITLE
Debugging generated images

### ISSUE THREAD

**Issue by bigaidream** [rahul99/msc_plan#10]
Devon said:
```
Hm. Though shouldn't it be without the graph information that the output should still look like cartoons? Perhaps just copying the input?

That the images look bad seems like a problem in itself. 
```

---

So, let's first remove the graph signals entirely to see if it can just copy the source images.

**Comment by rahul99** [comment]
1. Tried removing the graph signal entirely. Observed that input getting copied, but again with some noise. (updated results in dropbox.)

2. Also tried implementing custom cross-entropy loss using raw tensorflow api to achieve faster simultaneous training of D and G but it did not perform well.

3. Did some research on training GANs with input in similar appearance as output. We can try implementing stacked generator network that whose input is successive Generator outputs. Similar to the one in Liqian NIPS paper "Pose guided person image generation" and this one: https://github.com/hanzhanggit/StackGAN

**Comment by rahul99** [comment]
This is the current best result on GAN. Results are not optimal but still
satisfactory.

I think we should stop at this and move to part2 of the work. What do you
suggest?

In an hour I will write a short write up on what worked best and will
update Dropbox :)

Next would be graph construction on VG and model building on part 2 of the
paper.

Thanks,
Rahul
